### DISCUSSION TITLE
更新组织图标

### DISCUSSION THREAD

**Discussion by windingwind** [zotero-chinese/wiki#214] [社区建设]
应在Z7发布后更新组织图标。大家有什么好想法吗？

另可能需考虑新图标在尽量贴合Z图标风格及主要元素的同时，不直接对Z图标进行修改，以避免与Z本身图标的视觉形象侵权风险

**Comment by northword** [comment]
CC @zotero-chinese/all

**Comment by jiaojiaodubai** [comment]
我的想法比较草率：

- Z 7 的图标是一个夹着几片书签的便签本，上面写了红色的 Z。
- 类似地，保留原来的色彩搭配，突出地区特色，我们可以把便签本换成普通的文档图标或者卷轴之类，把红色的“Z”改成“中”或“Zh”。

**Reply by volatile-static** [reply]
支持Zh
